lib/wifi_user/use_case/email_signup.rb

```diff
@@ -25,11 +25,17 @@ def send_signup_instructions(email_address, login_details)
     client.send_email(
       email_address: email_address,
       template_id: credentials_template_id,
-      personalisation: login_details
+      personalisation: login_details,
+      email_reply_to_id: do_not_reply_template_id
     )
   end
 
   def credentials_template_id
     YAML.load_file("config/#{ENV['RACK_ENV']}.yml").fetch('notify_email_template_ids').fetch('self_signup_credentials')
   end
+
+  def do_not_reply_template_id
```

Comment:
I think `do_not_reply_email_address_id` (or similar) would be a better name. The `template_id` is the ID of the actual email that Notify sends.